admin-dev/themes/new-theme/scss/config/_settings.scss

```
@@ -14,7 +14,7 @@ $color-separator: #dfdfdf;
 $background-info: #dcf4f9;
 $purple: #34219e;
 $bright-status-text: #383838;
-$local-font: true;
+$local-font: false;
```

```suggestion
$local-font: true;
```

I know it's a tweak in order to use watch with material icons, but please don't push it (I need to look how to fix this..)

---

yep, sorry, I wasn't supposed to push it :(